pkg/color/formatter_test.go

```diff
@@ -0,0 +1,131 @@
+/*
+Copyright 2019 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package color
+
+import (
+	"bytes"
+	"io"
+	"testing"
+
+	"github.com/GoogleContainerTools/skaffold/testutil"
```

Comment:
hmm, we don't want to depend on skaffold here. can you copy the relevant methods from the `testutil` package into the container-structure-test codebase? they can live in another `testutil` package here.